episodes/08-quarto.Rmd

```diff
@@ -0,0 +1,122 @@
+---
+title: Getting started with Quarto (Optional)
+teaching: 30
+exercises: 15
+format:
+  html:
+    df-print: paged
+    keep-md: true
+source: md
+editor: 
+  markdown: 
+    wrap: 72
+---
+
+```{r}
+#| label: setup
+#| include: false
+source("data/download_data.R")
+```
+
+::: objectives
+-   Create a .qmd document containing R code, text, and plots
+-   Create a YAML header to control output
+-   Understand basic syntax of Quarto and Markdown
+-   Customise code chunks to control formatting
+-   Use code chunks and in-line code to create dynamic, reproducible
+    documents
+:::
+
+::: questions
+-   What is Quarto?
+-   How can I integrate my R code with text and plots?
+-   How can I convert .qmd files to .html?
+:::
+
+## Quarto
+
+Quarto is a flexible type of document that allows you to seamlessly
+combine executable R code, and its output, with text in a single
+document. These documents can be readily converted to multiple static
+and dynamic output formats, including PDF (.pdf), Word (.docx), and HTML
+(.html).
+
+The benefit of a well-prepared Quarto document is full reproducibility.
+This also means that, if you notice a data transcription error, or you
+are able to add more data to your analysis, you will be able to
+recompile the report without making any changes in the actual document.
+
+Quarto comes pre-installed with RStudio (as of v2022.07), so no action
+is necessary.
+
+<figure>
+
+```{r rmarkdown-wizards, echo = FALSE, fig.show = "hold", fig.alt = "R Markdown wizard monsters creating a R Markdown document from a recipe. Art by Allison Horst", out.width="100%"}
```

Comment:
We could use this image instead? https://cdn.myportfolio.com/45214904-6a61-4e23-98d6-b140f8654a40/c2ef45f1-4a68-4c7a-a43f-bdcf0f90e104_rw_1920.png?h=89c4f4138772e8b374e21a814cc91b2f

Reply:
Much better! Thanks!